composer.json

```diff
@@ -38,6 +38,6 @@
         }
     },
     "require-dev": {
-        "friendsofphp/php-cs-fixer": "^2.5"
+        "friendsofphp/php-cs-fixer": "^2.0"
```

Comment:
Was this intentional? Was this just so people can install a lower version if their dependencies require it?

Reply:
Yeah, that was the idea. :)